content/articles/new-plans.markdown

```diff
@@ -0,0 +1,114 @@
+---
+title: New plans and Legacy plans
+excerpt:
+categories:
+- Account
+---
+
+# New plans and Legacy plans
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+On June 2015 [we released one of our largedt upgrade to our platform](https://blog.dnsimple.com/2015/06/multiple-accounts-new-plans-and-new-features/), that introduced several new features such as multiple accounts, teams, and regional records. With the release of the multi-account feature we also changed our plans structure.
```

Comment:
`In` is more correct for a month. If we had a specific day, `On` would be correct.

A second note on this line is that `largedt` (in the link), should be `largest`.